src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2870,6 +2870,7 @@ public static void ChangeLocalTimeZone(string id)
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/69801", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Can you please try submitting this job without this ActiveIssue? This test doesn't fail on my machine. This might be an issue with your machine. Fiji did something weird to their time zone (#63476) and this test is testing that. But the time zone data comes from the OS, not from .NET. Your results are off by an hour which is what Fiji is changing so you might just have old time zone data for some reason.